spec/models/offer_spec.rb

```diff
@@ -9,5 +9,59 @@
 
       expect(offer.conditions.map(&:text)).to contain_exactly('Provide evidence of degree qualification', 'Do a backflip and send us a video')
     end
+
+    describe '#course_option' do
```

Comment:
Do we need to test model associations?

Reply:
I think in this instance we should, as the spec also acts as documentation for the class and the association is through another model.